distributions/demo-server/build.gradle

```diff
@@ -197,7 +197,7 @@ task installToolsAndDockerize {
 configurations.all {
     resolutionStrategy {
         eachDependency { DependencyResolveDetails details ->
-            if (details.requested.group == 'com.google.guava') {
+            if (details.requested.group == 'com.google.guava' && (details.requested.name != 'failureaccess' && details.requested.name != 'listenablefuture')) {
                 details.useVersion("${guavaVersion}")
                 details.because 'Some dependencies rely on lower version which does not support MoreCollections'
```

Comment:
I wonder if having this `resolutionStrategy` in `portability-transfer/build.gradle` would also have worked around the original problem?

This whole thing seems quite fragile.

Reply:
Hiya! sorry for the delay on this review, I was discussing this with zacsh@ yesterday and there are a few things that arent clear with this... 

1) where/how did you determine that you also needed `failureaccess` and `listenablefuture` here? Its not obvious from the logs 
2) If we delete this block entirely, would everything just work? 
3) Im looking through to see if there are other places where we explicitly make a new JacksonFactory, in theory there shouldn't be since each class should use the one that's registered in the extensioncontext in WorkerMain.java, but we've seen that the GooglePhotosInterface broke that :) so we should double check the project for other instances of this.

Reply:
If you remove them, you end up with failures to find those targets at the overridden version when you use the latest guava (which includes them and includes versioned references to them, and those versions get edited, IIRC).

Reply:
For 2; I don't know. I have definitely seen weird compilation failures related to be should-be-findable-but-aren't methods in `MoreCollections`. I don't understand why version resolution doesn't force a later version of Guava that has them naturally, but I haven't fully internalized gradle/Java's semantics here. I can try it later, but I'm OOO for the next 4d.

For 3, I ran a bulk editor in the monorepo that looked at `JacksonFactory` use, but it appears not to have found all the references in here, so I'll need to do a second pass manually.

Reply:
Had more time (and battery) than I thought, and I tried out removing the entire `resolutionStrategy` block, and that seems to have worked, so I'm going to roll that into the guava update commit.

Reply:
> For 3, I ran a bulk editor in the monorepo that looked at JacksonFactory use, but it appears not to have found all the references in here, so I'll need to do a second pass manually.

For context on this: we don't import the _entirety_ of this codebase into the monorepo (in case you have any other cases like ours in your lsc seeing only subsets of stuff).